src/Microsoft.Azure.WebJobs.Extensions.Storage/Queues/Config/QueuesOptionsSetup.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    internal class QueuesOptionsSetup : IConfigureOptions<QueuesOptions>
```

Comment:
If you want to avoid these tiny setup classes, you can use this if you want to configure inline with dependencies: https://twitter.com/davidfowl/status/1004248179628392448. 
But both ways work -- I know extra classes can be annoying sometimes.

Reply:
Funny - I spent a bit spelunking and looking online for a way to do that but didn't find it and wound up with what I have.